app/models/shipit/task.rb

```diff
@@ -228,6 +228,8 @@ def supports_rollback?
 
     def title
       definition.render_title(env)
+    rescue KeyError
+      'This task cannot be shown due to an incorrect variable name. Check your shipit.yml file'
```

Comment:
\nit I would include the name of the task instead of referring to `this` (if that is possible of course)

Reply:
I think I would have rescued in `render_title` itself.